doc/Security.xml

```diff
@@ -1094,6 +1094,11 @@
         providing authentication of users and authorizing access to both device services and
         external services, like cloud storage.</para>
       <para>OAuth 2.0 is a specification that describes how to issue access tokens. It is defined in RFC6749. OpenID Connect has been developed as an extension of OAuth 2.0.</para>
+      <para>OAuth 2.0 clients registration, as specified by RFC6749, for example sets up ClientId and
+        ClientSecret in addition to registering Authentication method listed in <xref
+          xmlns:xlink="http://www.w3.org/1999/xlink" linkend="table_smg_rzq_swb"/> and setting up
+        relevant metadata supporting client Authentication method, is a necessary prerequisite before
+        client is used in OAuth2.0 and OpenID Connect flows.</para>
```

Comment:
Not sure whether this sentence should go in the middle of a general OAuth2 introduction. How about moving it to the end of the paragraph and rephrase to improve readability:

  `For example OAuth 2.0 client registration, as specified by RFC6749 requires configuration of ClientId,
        ClientSecret, an authentication method according to <xref xmlns:xlink="http://www.w3.org/1999/xlink" linkend="table_smg_rzq_swb"/> plus server dependent supplemental parameters before the  client can use the OAuth2.0 and OpenID Connect flow.`

Reply:
2/15/2024 VEWG Telco
- Agree with the feedback, will update the PR accordingly.